packages/client/setup/routes.ts

```diff
@@ -67,7 +67,7 @@ export default function setupRoutes() {
   routes.push(
     {
       name: 'play',
-      path: '/:no',
+      path: '/:no(\\d+)',
```

Comment:
The route may not be a number, due to the [`routeAlias` option](https://sli.dev/custom/#per-slide-configuration)